src/sentry/static/sentry/app/views/onboarding/progress.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+import {onboardingSteps, stepDescriptions} from './utils';
+
+const ProgressNodes = React.createClass({
+  propTypes: {
+    params: React.PropTypes.object
+  },
+
+  steps: Object.keys(onboardingSteps),
+
+  inferStep() {
+    let {projectId} = this.props.params;
+    if (!projectId) return onboardingSteps.project;
+    return onboardingSteps.configure;
+  },
+
+  node(step, i) {
+    let nodeClass = classNames('node', {
+      done: i < this.inferStep(),
+      active: i === this.inferStep()
```

Comment:
Is `i` basically the current/active step? If so, could we name it something like `activeStep`?